ko/news/_posts/2023-06-29-redos-in-uri-CVE-2023-36617.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: news_post
+title: "CVE-2023-36617: ReDoS vulnerability in URI"
+author: "hsbt"
+translator: "shia"
+date: 2023-06-29 01:00:00 +0000
+tags: security
+lang: ko
+---
+
+ReDoS 취약점에 대한 보안 수정이 포함된 uri gem 버전 0.12.2, 0.10.3을 릴리스했습니다.
+이 취약점에는 CVE 식별자 [CVE-2023-36617](https://www.cve.org/CVERecord?id=CVE-2023-36617)이 할당되었습니다.
+
+## 세부 내용
+
+0.12.1 이전의 URI 구성 요소에서 ReDoS 문제가 발견되었습니다. URI 구문 분석기가 특정 문자가 포함된 유효하지 않은 URL을 잘못 처리합니다. 이로 인해 `rfc2396_parser.rb`와 `rfc3986_parser.rb`를 사용하는 URI 객체에 대한 문자열 구문 분석 실행 시간이 증가합니다.
```

Comment:
It seems that the parsing is string to URI, not URI to string. I think just switching two words is also ambiguous, so I edited little bit more.

```suggestion
0.12.1 이전의 URI 구성 요소에서 ReDoS 문제가 발견되었습니다. URI 구문 분석기가 특정 문자가 포함된 유효하지 않은 URL을 잘못 처리합니다. 이로 인해 `rfc2396_parser.rb`와 `rfc3986_parser.rb`를 사용해 문자열을 URI 객체로 구문 분석하는 데 걸리는 실행 시간이 증가합니다.
```

Reply:
+1 looks better